drivers/SmartThings/zigbee-button/src/zigbee-multi-button/robb/init.lua

```diff
@@ -0,0 +1,183 @@
+local device_management = require "st.zigbee.device_management"
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+local Basic = zcl_clusters.Basic
+local Level = zcl_clusters.Level
+local OnOff = zcl_clusters.OnOff
+local PowerConfiguration = zcl_clusters.PowerConfiguration
+local capabilities = require "st.capabilities"
+
+--[[
+The ROBB Wireless Remote Control has 4 or 8 buttons. They are arranged in two columns:
+
+All buttons on the left side support 'pressed' (OnOff > ON) and 'up_hold' (Level > MoveStepMode.UP).
+All buttons on the right side support 'pressed' (OnOff > OFF) and 'down_hold' (Level > MoveStepMode.DOWN).
+
+Each button-row represents one endpoint. The 8x remote control has four endpoints, the 4x remote control has two.
+That means each endpoint has two buttons.
+--]]
+
+local SWITCH_GROUP_CONFIGURE = "is_group_configured"
+local SWITCH8_NUM_ENDPOINT = 4
+local SWITCH8_NUM_BUTTONS = 8
+local SWITCH4_NUM_ENDPOINT = 2
+local SWITCH4_NUM_BUTTONS = 4
+
+local WIRELESS_REMOTE_FINGERPRINTS = {
+  { mfr = "ROBB smarrt", model = "ROB_200-007-0" },
+  { mfr = "ROBB smarrt", model = "ROB_200-008-0" }
```

Comment:
I would restructure this map to be more useful, and change the `CAN_HANDLE` appropriately.

```
local ROBB_MFR_STRING = "ROBB smarrt"
local WIRELESS_REMOTE_FINGERPRINTS = {
  "ROB_200-007-0" = { endpoints = 2, buttons = 4},
  "ROB_200-008-0" = { endpoints = 4, buttons = 8}
}
```

this way, in other places where you're doing this:

`number_of_buttons = device:get_model() == "ROB_200-007-0" and SWITCH8_NUM_BUTTONS or SWITCH4_NUM_BUTTONS`

you can just do

`number_of_buttons = WIRELESS_REMOTE_FINGERPRINTS[device:get_model()].buttons`

Reply:
solved with 714d055